ONSdigital/eq-author-app
# Ear-2005-add-picker-and-results-table

### What is the context of this PR?

This PR is linked to this [Jira ticket](https://jira.ons.gov.uk/browse/EAR-2005)

### How to review

-Checkout `EAR-2005-add-picker-and-results-table`
-Enable the `dataset` feature flag


- Open local author and navigate to the data page then ONS dataset:
  - ensure a select component shows with Survey ID as the default option
  - click one of the survey id's
  - ensure that the datasets table shows the survey id you selected piped into the title
  - ensure this works for other survey id's
  - set the survey id to the default option and ensure that the table collapses
